metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/operators/datahub.py

```diff
@@ -60,4 +66,36 @@ def __init__(  # type: ignore[no-untyped-def]
         self.metadata = mces
 
     def execute(self, context):
```

Comment:
does it make sense to have this behavior be opt-in or opt-out? if definitely should be configurable, but I'm not sure what the norms are with airflow

Reply:
I have checked the other operators, but there is no opt-in/opt-out implementation; hence for now I have also skipped. If we want then by default we can keep it as `True` and in case of avoid template rendering we can pass additional parameter while creating instance of the DatahubEmitterOperator